packages/unchained-client/src/types.ts

```diff
@@ -28,9 +30,10 @@ export interface Trade {
   type: TradeType
 }
 
+// these are user facing values, and should be rendered as such
```

Comment:
same as above

Reply:
Yeah, these are not only user-facing values. We use them in web to render the relevant transaction type's component and they have a different casing than the similarly-named `chain-adapters` enum, which results in ZRX, Thor, and, with this diff, CowSwap Txs falling back to the generic (unknown) Tx component.

Reply:
So `TransferType`, `TradeType` and `Status` were all types that lived in unchained initially and we created chain adapters types at the time so we weren't passing back end unchained types through to web. This has changed a bit now that we have the parser in lib and we can probably clean up these types so we just have one reference of them to use for both chain adapters and unchained client

Reply:
But yes, they are all values for the client to render as necessary ultimately